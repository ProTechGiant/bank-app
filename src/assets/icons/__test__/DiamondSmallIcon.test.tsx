import { create } from "react-test-renderer";

import { DiamondSmallIcon } from "../DiamondSmallIcon";

describe("icon/DiamondSmallIcon", () => {
  it("renders a DiamondSmallIcon", () => {
    const result = create(<DiamondSmallIcon />);

    expect(result).toMatchSnapshot();
  });
});
