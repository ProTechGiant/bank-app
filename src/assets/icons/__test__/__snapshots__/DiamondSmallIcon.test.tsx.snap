// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/DiamondSmallIcon renders a DiamondSmallIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="9"
  bbWidth="10"
  fill="none"
  focusable={false}
  height={9}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 9,
        "width": 10,
      },
    ]
  }
  vbHeight={9}
  vbWidth={10}
  width={10}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M8.5 0H1.5L0 3L5 9L10 3L8.5 0ZM3.81 2.5L4.56 1H5.44L6.19 2.5H3.81ZM4.5 3.5V6.84L1.72 3.5H4.5ZM5.5 3.5H8.28L5.5 6.84V3.5ZM8.63 2.5H7.305L6.555 1H7.88L8.63 2.5ZM2.12 1H3.445L2.695 2.5H1.37L2.12 1Z"
      fill={
        {
          "payload": 4294638330,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
